api/envoy/extensions/filters/network/generic_proxy/action/v3/action.proto

```diff
@@ -16,7 +16,7 @@ import "validate/validate.proto";
 option java_package = "io.envoyproxy.envoy.extensions.filters.network.generic_proxy.action.v3";
 option java_outer_classname = "ActionProto";
 option java_multiple_files = true;
-option go_package = "github.com/envoyproxy/go-control-plane/contrib/envoy/extensions/filters/network/generic_proxy/action/v3;actionv3";
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/generic_proxy/action/v3;actionv3";
```

Comment:
cc @htuch will this impact go based control planes that provided a contrib?
IMHO as this is contrib it may be ~ok to break, but just wondering whether we should reconsider the impact of moving an extension from contrib to extensions in the future.

Reply:
I'm not worried about that since it was contrib (it will have build consequences though).